k8s/upgrades/0.4.0-0.5.0/controller.patch.yml

```diff
@@ -0,0 +1,44 @@
+{
```

Comment:
can we name this file as controller.patch.tpl.yml

Reply:
Done !